### PR TITLE
[#1141] change information in service details

### LINKED ISSUE #1141: Change information (content and naming) which we display in service details
**Is your feature request related to a problem? Please describe.**
Now, in the service details page, we display information about the service which is more or less inappropriate. Moreover, we display the same information for orderable and open access service but this attitude is wrong. 
Fields names also are inconsistent, for eg. for open access services we display "Order date" but in fact, these services aren't orderable.

**Describe the solution you'd like**

For  internal ordering services
- in the table display:
- [x] Service name : name with link to the service in MP
- [x] Service offer :  service offer name( if there is no service offer, do not show this field)  
- [x] Order date : to show order date 
- [x] Service access: Internal ordering ( link to FAQ) 
- [x] Project name : link to the project name 
- [x] SLA : link from the front page in the service view ( if there is no SLA, do not show this field)  
- [x] Provider : link to the Provider name

- remove form the table:
- [x] Status field from Service Details 

- under 1. from above picture display:
- [x] ORDER STATE: display status as before

For open access services 
- display:

- [x] Service name : name with link to the service in MP
- [x] Service offer : service offer name ( if there is no service offer, do not show this field)  
- [x] Order date : to show date when service was added to the Project 
- [x] Service access: open ( link to FAQs) 
- [x] Project name : link to the project name 
- [x] SLA : link from the front page in the service view ( if there is no SLA, do not show this field)  
- [x] Provider : link to the Provider name

- remove:
- [x] Status field from Service Details 

- under 1. from above picture display:
- [x]  Go to service (External link with service website) 

For external ordering type
- display:

- [x] Service name : name with link to the service in MP
- [x] Service offer : service offer name ( if there is only one offer, do not show this field)  
- [x] Order date : to show date when service was added to the Project 
- [x] Service access: External ordering ( link to FAQs) 
- [x] Project name : link to the project name 
- [x] SLA : link from the front page in the service view ( if there is no SLA, do not show this field)  
- [x] Provider : link to the Provider name

- remove:
- [x] Status field from Service Details 

- under 1. from above picture display:
- [x] ORDERABLE EXTERNALLY (External link with service website) 

Additionally to change Tabs name : 

- [x] from SERVICE DETAILS title to DETAILS
- [x] from CONTACT WITH PROJECT SUPPORT to ODER HISTORY (additional tab with "contact with service provider" will be created in a different ticket) - to be shown only to normal services type. 

**additional context**
If exist we leave section:
TECHNICAL PARAMETERS
VOUCHER
RATING

### PR DESCRIPTION
**Pull request by goreck888**
Change information in the ordered/added to project service in the project item view.

Fixes #1141

### CODE REVIEW

File: app/javascript/controllers/pages_controller.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Controller } from "stimulus"
```
Comment (mkasztelnik) [inline app/javascript/controllers/pages_controller.js]: This should go to the separate PR since this is not connected with the rest of the PR.

File: app/views/projects/services/_details.html.haml [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
-  %dd
-    %span.text-uppercase{ class: "status-#{project_item.status}" }
-      = project_item.status
+  %dt Service access:
+  %dd= link_to t("project_item.offer_type.#{project_item.offer.offer_type}"),
+  "#{page_path("help")}#help-services-service-types"
 %dl
   %dt
-    Project:
+    Project name:
```
Comment (mkasztelnik) [inline app/views/projects/services/_details.html.haml]: @agpul, @MinLidia do we need this row since the service project is selected on the left?

File: app/views/projects/services/_details.html.haml [hunk 3]
```diff
@@ -1,18 +1,25 @@
...
   %dd= link_to project.name, project
+- if project_item.service.sla_url.present?
+  %dl
+    %dt SLA:
+    %dd= link_to "URL", project_item.service.sla_url
```
Comment (mkasztelnik) [inline app/views/projects/services/_details.html.haml]: Maybe a little bit more verbose name here, e.g.: "Service Level Agreement"

File: app/views/projects/services/_layout.html.haml [hunk 4]
```diff
@@ -3,24 +3,29 @@
...
-        %p.text-uppercase.text-right.font-weight-bold.text-secondary.mb-0
-          %span{ class: "status-#{project_item.status}" }= project_item.status
+      .col-md-4.float-right.justify-content-center
+        - if project_item.orderable?
+          %p.text-uppercase.text-right.font-weight-bold.text-secondary.mb-0
+            order state:
+            %span{ class: "status-#{project_item.status}" }= project_item.status
+        - elsif !project_item.offer.webpage.blank?
+          = link_to t("service.#{project_item.offer.offer_type}.link"), project_item.offer.webpage,
+          class: "btn btn-primary float-right", target: "_blank", title: "External link"
```
Comment (mkasztelnik) [inline app/views/projects/services/_layout.html.haml]: I'm always frustrated when a webpage is opening new tab for me.  I remember that we already had similar conversation with @agpul and @abacz and the conclusion was that we should avoid `target: "_blank"` when possible - is it the exception and we are sure that new tab should be opened here?

File: app/views/projects/services/_details.html.haml [hunk 5]
```diff
@@ -1,18 +1,25 @@
...
 %dl
-  %dt Providers:
-  %dd= service_providers_list(project_item)
-%dl
-  %dt Status:
-  %dd
-    %span.text-uppercase{ class: "status-#{project_item.status}" }
-      = project_item.status
+  %dt Service access:
+  %dd= link_to t("project_item.offer_type.#{project_item.offer.offer_type}"),
```
Comment (mkasztelnik) [inline app/views/projects/services/_details.html.haml]: Maybe we can add some tooltip explaining service type in one sentence?

File: app/views/projects/services/_details.html.haml (hunk 3 above)
Comment (agpul) [reply app/views/projects/services/_details.html.haml]: I agree with that I propose to use Service Level Agreements statement instead of URL

File: app/views/projects/services/_details.html.haml (hunk 5 above)
Comment (agpul) [reply app/views/projects/services/_details.html.haml]: Tooltip seems like a good idea.

Comment (MinLidia) [reply app/views/projects/services/_details.html.haml]: Not sure if we need to have additional Tooltip. Now if you click on the link it opens you the page with description of all 3 types in 1 sentence.
In case you see a need in tooltip, then I suggest to delete link to FAQ and make the description taking text from FAQ. Now each access has a description in one sentence. 
Screenshot 2019-12-05 at 10 59 10

File: app/views/projects/services/_details.html.haml (hunk 2 above)
Comment (MinLidia) [reply app/views/projects/services/_details.html.haml]: I suggest to leave Project name. In cases when user was redirected to the offer from e-mails, he would not know what project it was / remember. And if he has more than 2 projects, it is not really visible that he is in the below project. 
Screenshot 2019-12-05 at 11 06 18

File: app/javascript/controllers/pages_controller.js (hunk 1 above)
Comment (goreck888) [reply app/javascript/controllers/pages_controller.js]: Ok, this is separate commit, so i can make separate PR for this

File: app/views/projects/services/_details.html.haml (hunk 5 above)
Comment (goreck888) [reply app/views/projects/services/_details.html.haml]: Tooltip added

File: app/views/projects/services/_layout.html.haml (hunk 4 above)
Comment (goreck888) [reply app/views/projects/services/_layout.html.haml]: I think yes, because I don't know if user would to make something else in mp, and link to service on the same tab disable mp. But we can consult it on monday.

File: app/views/projects/services/_details.html.haml (hunk 3 above)
Comment (goreck888) [reply app/views/projects/services/_details.html.haml]: Done

File: app/views/projects/services/_layout.html.haml [hunk 6]
```diff
@@ -3,28 +3,31 @@
...
               %i.fas.fa-chevron-left.fa-xs.mr-1
               = link_to "back to #{project.name} project services", [project, :services]
-      .buttons-panel.col-md-3.float-right.d-flex.justify-content-center
-        %p.text-uppercase.text-right.font-weight-bold.text-secondary.mb-0
-          %span{ class: "status-#{project_item.status}" }= project_item.status
+      .col-md-4.float-right.justify-content-center
+        - if project_item.orderable?
+          %p.text-uppercase.text-right.font-weight-bold.text-secondary.mb-0
+            %span{ class: "status-#{project_item.status}" }= project_item.status
+        - elsif !project_item.offer.webpage.blank?
```
Comment (martaswiatkowska) [inline app/views/projects/services/_layout.html.haml]: Maybe -elsif project_item.offer.webpage.present? ?

Comment (goreck888) [reply app/views/projects/services/_layout.html.haml]: Ok
